Documentation/In-depth/TheInstallTool/Index.rst

```diff
@@ -192,4 +207,3 @@ and displays hints if something goes wrong.
    TYPO3 might have problems executing it. If the command does not work, you
    might need to adjust the settings (via "Settings" > "Configuration Presets"
    > "Image Handling").
-
```

Comment:
Please keep the empty line at EOF

Reply:
@mbrodala if you hit the edit mode there is an empty line at the end of the file. Somehow this sometimes appears in PRs, but the empty line at the end is automatically added so it didn't disapear.